static/sass/custom/_series_tags.scss

```diff
@@ -0,0 +1,22 @@
+%series-tag {
+  display: inline-block;
+  border: 1px solid $color-mid-dark;
+  border-radius: 4px;
+  font-size: 0.875rem;
+  font-weight: 500;
+  color: $color-mid-dark;
+  padding: 0 0.5rem;
+  margin-right: 0.25rem;
+  margin-bottom: 0.25rem;
+  text-transform: capitalize;
+}
+
+.series-tag {
+  @extend %series-tag;
```

Comment:
I would add the default `border-color` and `color` here and remove it from the placeholder to reduce overriding styles in the `.series-tag--kubernetes` pattern